src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf

```diff
@@ -438,6 +438,10 @@
                 <source>This value is not a valid MAC address.</source>
                 <target>Ta wartość nie jest prawidłowym adresem MAC.</target>
             </trans-unit>
+            <trans-unit id="113">
+                <source>This URL does not contain a TLD.</source>
+                <target>Podany adres URL nie zawiera TLD.</target>
```

Comment:
Let's not assume people know what a TLD is? GPT suggests this :)
`Ten URL nie zawiera domeny najwyższego poziomu (TLD).`

Reply:
Sure, I didn't see other translations expanding the meaning of either URL or TLD, I'll update the PR shortly. :)

Reply:
@nicolas-grekas done!